src/VisualStudio/Core/Def/Storage/AbstractCloudCachePersistentStorageService.cs

```diff
@@ -38,7 +33,7 @@ protected sealed override bool ShouldDeleteDatabase(Exception exception)
         }
 
         protected sealed override async ValueTask<IChecksummedPersistentStorage?> TryOpenDatabaseAsync(
-            SolutionKey solutionKey, string workingFolderPath, string databaseFilePath, CancellationToken cancellationToken)
+            IPersistentStorageConfiguration configuration, SolutionKey solutionKey, string workingFolderPath, string databaseFilePath, CancellationToken cancellationToken)
```

Comment:
config moved from 'state' about the service, to parameters passed into work it does (which makes a lot more sense to me IMO).

Reply:
The config is only a test hook. The default implementation is the only real impl. It's only replaced for testing. As such it should never be different between calls to the service.